### DISCUSSION TITLE
Upgrading to version 4.6 from 3.5 getting a vite warning in console

### DISCUSSION THREAD

**Discussion by mauricioxrodriguez** [mozilla/pdf.js#18841] [Q&A] [answered]
After upgrading to pdfjs-dist version 4.6 I get the following vite warning in vs code console from file: 
.angular/cache/18.0.1/vite/deps/pdfjs-dist_web_pdf_viewer__mjs.js
5964|        const sandbox = import(
5965|          /*webpackIgnore: true*/
5966|          sandboxBundleSrc
   |          ^^^^^^^^^^^^^^^^
5967|        );
5968|        sandbox.then((pdfjsSandbox) => {
The above dynamic import cannot be analyzed by Vite.
See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations for supported dynamic import formats. If this is intended to be left as-is, you can use the /* @vite-ignore */ comment inside the import() call to suppress this warning.

**Comment by Snuffleupagus** [comment, marked as answer]
Please see PR #18395, which never landed since there was no real demand.
